definitions/features/upstream.rb

```diff
@@ -8,7 +8,59 @@ class Features::Upstream < ForemanMaintain::Feature
     end
   end
 
-  def setup_repositories(_version)
-    raise NotImplementedError
+  def setup_repositories(version)
+    activation_key = ENV['EXTERNAL_KATELLO_ACTIVATION_KEY']
+    org = ENV['EXTERNAL_KATELLO_ORG']
```

Comment:
Organization's are in Foreman, so I would proffer we either call it `_ORG` or `_FOREMAN_ORG`.

What does the `EXTERNAL_` imply in this context? external to what?

Reply:
I thought keeping EXTERNAL should help to avoid self registrations?